force-app/main/default/classes/EmailUtilityTest.cls

```diff
@@ -0,0 +1,57 @@
+@isTest
+private class EmailUtilityTest {
+    @testSetup static void setupRecords() {
+ 		Contact con = new Contact(LastName='TestEmail', Email='vip.nepochelovich@gmail.com');
+        Contact conWithoutEmail = new Contact(LastName='conWithoutEmail');
+        insert con;
+        insert conWithoutEmail;
+        Lead myLead = new Lead(LastName = 'Fry', Company='Fry And Sons');
+        insert myLead;
```

Comment:
try to separate some blocks of code with line space for better understanding.
ex:
Contact conWithoutEmail = new Contact(LastName='conWithoutEmail');
insert conWithoutEmail;

Lead myLead = new Lead(LastName = 'Fry', Company='Fry And Sons');
insert myLead;

Database.LeadConvert lc = new Database.LeadConvert();
lc.setLeadId(myLead.id);
LeadStatus convertStatus = [SELECT Id, MasterLabel FROM LeadStatus WHERE IsConverted=true LIMIT 1];
lc.setConvertedStatus(convertStatus.MasterLabel);
lc.isDoNotCreateOpportunity();
lc.setOpportunityName('NewOpportunity');